spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get users_path
+      expect(response.body).to include('list of all users')
```

Comment:
- Good job done so far, this test keeps failing, this is because the placeholder displayed in your view file is `List of users`. Please fix this and ensure all tests pass 😄